packages/search-ui-app-search-connector/src/__tests__/responseAdapter.test.js

```diff
@@ -8,6 +8,46 @@ describe("adaptResponse", () => {
   it("adapts empty response", () => {
     expect(adaptResponse(emptyResponse)).toEqual(adaptedEmptyResponse);
   });
+
+  it("will accept additional facet value fields to inject into response", () => {
+    const options = {
+      additionalFacetValueFields: {
+        location: {
+          center: "73.102, -78.120",
+          unit: "u"
+        }
+      }
+    };
+
+    expect(adaptResponse(response, options).facets.location).toEqual([
+      {
```

Comment:
I don't feel strongly either way, but I'm seeing most of these objs being defined below as `const`s - do we want to follow that pattern as well and define this as `adaptedLocationResponse` or something similar as well as moving options down as well (maybe as `geoOptions`)? 🤷‍♀

(TBH I'm fine with this as-is, although I'll confess the variable hoisting from the bottom of the file is a bit confusing.)

Reply:
I frankly have no opinion on this, so just I'll update it.

Reply:
Except for the hoisting, that's probably a bad practice.